src/bowling2.js

```diff
@@ -0,0 +1,53 @@
+class Bowling {
+
+    constructor(){
+        
+        this.scoreCard = []
+    
+        }
+
+  
+
+    roll(Roll) {
+       
+        this.scoreCard.push(Roll)
+      
+    } 
+
+    calculateScore(rolls) {
+        var rollCount = 0 
+        let score = 0
+        
+        for( let frameCount = 0; frameCount < 10; frameCount++) {
```

Comment:
This seems too convoluted, it might be a bit simpler to store the scores in frames before making this calculation.